betterlockscreen

```diff
@@ -967,4 +977,4 @@ done
 [[ $runlock ]] && lockselect "$lockstyle" && \
     { [[ $runsuspend ]] && systemctl suspend; }
 
-exit 0
\ No newline at end of file
+exit 0
```

Comment:
There seems to be no real change please revert this

Reply:
this appears to be the diff complaining about no newline at eof in the source (req'd for posix compliance in a text file)

Reply:
ok, fixed!
weird technicality :)